src/features/MentorPage/components/MentorList/MentorCard/List/index.tsx

```diff
@@ -37,13 +42,13 @@ export const ListCard: React.FC<Props> = ({ setVisibleCard, mentor }) => {
   );
 };
 
-const Container = styled.div<{ isMobile: boolean }>`
+const Container = styled.div<{ isHomePage: boolean; isMobile: boolean }>`
   background-color: ${palette.white};
   border-radius: 0.75rem;
   display: flex;
   flex-direction: column;
 
-  ${({ isMobile }) =>
+  ${({ isHomePage, isMobile }) =>
```

Comment:
Now we have introduced a nested ternary here. I would like to see a better scalable solution here. For example now we have options
_isMobile_, _isHomePage_, and default. 

Maybe there will be _isHomePageAndMobile_? How will the styles be applied then?
Could we use a record? Or applying styles inline with &&?

Reply:
Yeah we don't want those. I added the `no-nested-ternary` Eslint rule and fixed this and the other errors that appeared.